src/prowjobsscraper/equinix_usages.py

```diff
@@ -0,0 +1,63 @@
+import logging
+from datetime import datetime
+from typing import Final, Optional
+
+import requests
+from pydantic import BaseModel
+
+logger = logging.getLogger(__name__)
+
+
+class EquinixUsage(BaseModel):
+    description: Optional[str]
+    facility: str
+    metro: str
+    name: str
+    plan: str
+    plan_version: str
+    price: float
+    quantity: float
+    total: float
+    type: str
+    instance: Optional[str]
+    unit: str
+    start_date: datetime
+    end_date: Optional[datetime]
+
+    @property
+    def job_build_id(self) -> str:
+        return self.name.split("-")[-1]
```

Comment:
check the boundaries, someone (like me :see_no_evil:) may spawn an instance for a test and not respect the naming convention.

Reply:
currently in that case build_id is set to the whole name, how would you handle it differently?

Reply:
you're right, it's fine. Can you add a comment in the code to explain this behavior is intended?